### PR TITLE
Rename package to co.ello.ElloApp

### PR DESCRIPTION
**Pull request by steam**
`ello.co.ello` was not a good package name 😏

### CODE REVIEW

**Comment by steam** [comment]
ping @jayzes

File: app/src/main/java/co/ello/ElloApp/MainActivity.java [hunk 1]
```diff
@@ -33,8 +33,8 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-        mSwipeLayout = (SwipeRefreshLayout) findViewById(R.id.container);
+        setContentView(co.ello.ElloApp.R.layout.activity_main);
+        mSwipeLayout = (SwipeRefreshLayout) findViewById(co.ello.ElloApp.R.id.container);
```
Comment (colinta) [inline app/src/main/java/co/ello/ElloApp/MainActivity.java]: These have to be there?  I thought that having `package` at the top made this prefix assumed.

**Comment by colinta** [comment]
LGTM!  Though I would try it without the `co.Ello.ElloApp.` prefix on `R`.

**Comment by steam** [comment]
Good catch @colinta, I hadn't noticed that those had the package prefix. Removed now.

**Comment by colinta** [comment]
PR reviews FTW!

**Comment by jayzes** [comment]
:metal:

**Pull request merged**